### ISSUE TITLE
Generate an intermediate dataset that turn the task into a tagged-text multi-label classification

### ISSUE THREAD

**Issue by tagny** [lengocluyen/relation_extraction_textmine25#16]
The principle is to mark:
- add entity mention tags as special tokens to the pretrained BERT-base model to fine-tune: `tokenizer.add_tokens(['<e1>', '</e1>', '<e2>', '</e2>'], special_tokens=True)`
- the mentions of the first entity of the relation between special tokens `<e1>` and `</e1>`
- the mentions of the second entity of the relation between special tokens `<e2>` and `</e2>`

then we get a corpus where for each sentence, we generate a new data for each possible ordered pair of entities that contains:
- as input: the sentence augmented with the marked entity mentions
- as output we have the probability `p` of each possible relation `r(e1, e2)`:
    - `p=1` if `r(e1, e2)` is in the ground truth
    - `p=0` otherwise

e.g.  `"<e1><CIVILIAN>Barack Obama</e1> is in <e2><PLACE>Paris</e2>",  {IS_LOCATED_IN}`
